src/TransactionController.ts

```diff
@@ -358,7 +409,7 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 	 *
 	 * @returns - Promise resolving when this operation completes
 	 */
-	async queryTransactionStatuses() {
+	async queryTransactionStatuses(): Promise<any> {
```

Comment:
This return type should be `Promise<void>`, not `<any>. Also note that an asynchronous function has an implied return type already, so you can remove this return type entirely. This only needs to be expressed if you resolve the promise with a specific non-void type, like `Promise<string>`.